nodejs/week4/homework-template/src/frontend/pages/home.js

```diff
@@ -0,0 +1,59 @@
+const searchSpecificMeal = event => {
+	const searchedTitle = this.event.target.value;
+	fetch(`/api/meals?title=${searchedTitle}`)
```

Comment:
This would ensure that the query parameter is encoded correctly:

```suggestion
	fetch(`/api/meals?title=${encodeURIComponent(searchedTitle)}`)
```